pkg/client/ocm/client.go

```diff
@@ -496,31 +514,43 @@ func (c client) GetQuotaCosts(organizationID string, fetchRelatedResources, fetc
 		return nil, err
 	}
 	quotaCostList := quotaCostResp.Items()
-	return quotaCostList, nil
+
+	if !fetchRelatedResources || len(relatedResourceFilters) == 0 {
+		return quotaCostList.Slice(), nil
+	}
+
+	var quotaCosts []*amsv1.QuotaCost
```

Comment:
Have we verified the behavior is the same when there are no results with this change? is a nil slice returned? an empty one?

Reply:
A nil slice is returned. yes it's the same, i tested it manually though but i'll add a test for this.